get.go

```diff
@@ -200,7 +200,9 @@ func get(ctx context.Context, logger *log.Logger, c getConfig, rawTarget string)
 			if err != nil {
 				return errors.Wrapf(err, "found unparsable mod file %v. Uninstall it first via get %v@none or fix it manually.", e, name)
 			}
-			defer errcapture.Do(&err, mf.Close, "close")
+			if err := mf.Close(); err != nil {
```

Comment:
Thanks! Can we ensure we don't use `mf` variable after Close? Here we can do easily with `dpkg := *mf.DirectPackage()` and then close and then append to target (: